### ISSUE TITLE
Backup as XML

### ISSUE THREAD

**Issue by isarrider** [michaldrabik/showly#453]
It would be nice to be able to export as an XML...
That way it is also human readable...
Thoughts?

**Comment by fabianski7** [comment]
definitely. it is very useful when showly is used without some service like trakt.

**Comment by martin78cz** [comment]
I agree. Easy to read export would be great. I'm adding my vote for this feature.

**Comment by tonydero** [comment]
Another vote for a human-readable file export option here since this one is open and #26 was closed a while back with no word. Love this app, but an export option is really needed without having to use another service.

**Comment by XxTriviumxX** [comment]
can you just, like, make it happen?  doesn't have to be xml though. Anything to back up the app's data?

**Comment by blackholeearth** [comment]
backup restore as json file exists now.

can close this.
